tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -67,9 +67,9 @@ class DuckDBPyConnection:
     def from_csv_auto(self, file_name: str) -> DuckDBPyRelation: ...
     def from_df(self, df: pandas.DataFrame = ...) -> DuckDBPyRelation: ...
     @overload
-    def from_parquet(self, file_glob: str, binary_as_string: bool = ..., *, file_row_number: bool = ..., filename: bool = ..., hive_partitioning: bool = ...) -> DuckDBPyRelation: ...
+    def from_parquet(self, file_glob: str, *, binary_as_string: bool = ..., file_row_number: bool = ..., filename: bool = ..., hive_partitioning: bool = ..., union_by_name: bool = ...) -> DuckDBPyRelation: ...
```

Comment:
Can you move the stars in these signatures back to the original correct location?

Reply:
Fixed